pom.xml

```diff
@@ -96,6 +96,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.3.0</version>
```

Comment:
I would also remove the version of the standard maven plugins.
The current versions are inherited by the maven super pom. if there is no reason to fix them to a certain version.

Reply:
maven is complaining about it during the build, so it seems that it needs/wants it. I don't know if it is really useful or not